src/components/venueSignin.js

```diff
@@ -27,49 +26,49 @@ export default class SignIn extends Component {
      this.setVenue(res);
    })
   }
-  
+
   validSignIn(res){
-    if( res != undefined){
+    if( res !== undefined){
       this.setVenue(res[0])
     }
     else{
       //error
     }
   }
+
    setVenue = (res) => {
-    this.setState({
-      venueSignedIn : res[0]
```

Comment:
notice a swap from res[0] to res

Reply:
Aye - res[0] is passed in as an argument when setVenue() is called, so it when it arrives res is the first from the array!